pyfms/py_fms/_functions.py

```diff
@@ -0,0 +1,25 @@
+from ctypes import POINTER, c_char_p, c_int
```

Comment:
Is this the system ctypes or the pyfms package ctypes?

Reply:
It's the system ctypes.